src/applications/caregivers/actions/fetchMapBoxGeocoding.js

```diff
@@ -25,13 +25,13 @@ export const fetchMapBoxBBoxCoordinates = async (query, client = null) => {
     })
     .send()
     .then(({ body: response }) => {
-      if (!response.features[0].bbox) {
+      if (!response.features.length) {
```

Comment:
🎨 does this support `.any?` / `.none?`? I'd opt for the more direct readability if they work:
```suggestion
      if (response.features.none?) {
```

Reply:
I don't think those exist in javascript. If this was on the backend we'd have some nice helpers to make it more readable :)

Reply:
Argh, rubybrain, you've failed me!

yep, never mind